ch04.adoc

```diff
@@ -310,34 +317,38 @@ Negative years are invalid in time coordinates and reference datetimes in the **
 In the **`standard`** calendar, 1582-10-15 0:0:0 is exactly 1 day later than 1582-10-4 0:0:0.
 Therefore datetimes in the range from (and including) 1582-10-5 0:0:0 until (but excluding) 1582-10-15 0:0:0 are invalid, and must not be used as reference in **`units`**.
 It is recommended that a reference datetime before the discontinuity should not be used for datetimes after the discontinuity, and vice-versa.
++
+__The **`standard`** calendar is used both for (mostly observational) data with real-world datetimes, including leap seconds (after 1972), and for data not including leap seconds__, generated by models according to the Gregorian calendar.
```

Comment:
Not sure if this emphsis is required, given that the following sentence is emphasised.
```suggestion
The **`standard`** calendar is used both for (mostly observational) data with real-world datetimes, including leap seconds (after 1972); and for data not including leap seconds, generated by models according to the Gregorian calendar.
```

Reply:
I wanted to emphasise first that the standard calendar is used (even though we may not like it) both for data with leap seconds and for data without leap seconds. The next emphasis is the caution, which is a consequence of it.